src/components/proofs/ContractListContainer.tsx

```diff
@@ -8,5 +10,10 @@ const container = classnames(
 )
 
 export default function ({ children }: ChildrenProp) {
-  return <div className={container}>{children}</div>
+  const listRef = useAutoAnimate({})
+  return (
+    <div className={container} ref={listRef as Ref<HTMLDivElement>}>
```

Comment:
Can't we do `const listRef = useAutoAnimate<HTMLDivElement>({})`?

Reply:
No, we can't, it doesn't accept type notations